zips/zip-0246.rst

```diff
@@ -247,9 +246,10 @@ For each Action Group, the following elements are included in the hash::
 
     T.4a.i   : orchard_actions_compact_digest      (32-byte hash output)
     T.4a.ii  : orchard_actions_noncompact_digest   (32-byte hash output)
-    T.4a.iii : flagsOrchard                        (1 byte)
-    T.4a.iv  : anchorOrchard                       (32 bytes)
-    T.4a.v   : nAGExpiryHeight                     (4 bytes)
+    T.4a.iii : orchard_burn_digest                 (32-byte hash output)
+    T.4a.iv  : flagsOrchard                        (1 byte)
+    T.4a.v   : anchorOrchard                       (32 bytes)
+    T.4a.vi  : nAGExpiryHeight                     (4 bytes)
```

Comment:
This should follow the same ordering as in the transaction, so if the burn information is positioned after `nAGExpiryHeight`:
```suggestion
    T.4a.iii : flagsOrchard                        (1 byte)
    T.4a.iv  : anchorOrchard                       (32 bytes)
    T.4a.v   : nAGExpiryHeight                     (4 bytes)
    T.4a.vi  : orchard_burn_digest                 (32-byte hash output)
```